tests/cross/src/test/scala/tests/pc/InlayHintsSuite.scala

```diff
@@ -1075,6 +958,21 @@ class InlayHintsSuite extends BaseInlayHintsSuite {
        |  def foo(x: => Int, y: Int, z: => Int)(w: Int, v: => Int): Unit = ()
        |  foo(/*=> */1, 2, /*=> */3)(4, /*=> */5)
        |}
+       |""".stripMargin,
+    namedParameters = false
+  )
+
+  check(
+    "by-name-regular-named",
+    """|object Main{
+       |  def foo(x: => Int, y: Int, z: => Int)(w: Int, v: => Int): Unit = ()
+       |  foo(1, 2, 3)(4, 5)
+       |}
+       |""".stripMargin,
+    """|object Main{
+       |  def foo(x: => Int, y: Int, z: => Int)(w: Int, v: => Int): Unit = ()
+       |  foo(/*x = => */1, /*y = */2, /*z = => */3)(/*w = */4, /*v = => */5)
```

Comment:
@harpocrates I noticed that our PR interact with each other, what do you think about the mix of named parameters and by name ones? Alternatively, we could drop '=' and have:
```suggestion
       |  foo(/*x => */1, /*y = */2, /*z => */3)(/*w = */4, /*v => */5)
```

Reply:
I think seeing `= =>` is probably the most reasonable thing - unfortunately in scala `x => 1` starts looking like a lambda w/ arg `x`. I don't have a strong opinion though...